dist/message.js

```diff
@@ -0,0 +1,70 @@
+var RiseVision = RiseVision || {};
+RiseVision.Common = RiseVision.Common || {};
+
+RiseVision.Common.Message = function (mainContainer, messageContainer) {
+  "use strict";
+
+  var _active = false;
+
+  function _init() {
+    try {
+      messageContainer.style.height = mainContainer.style.height;
```

Comment:
@alex-deaconu `mainContainer` is intended to be the the container `div` in the widgets that contains all the main content of the widget. It's passed as a param so that the height can be referenced but also so it can toggle its visibility, see lines 31 & 45. If I was to retrieve the element in this module, then the ids of this `mainContainer` element would have to be consistent in all the Widgets (they may not be).